src/renderer/gdi/state.cpp

```diff
@@ -374,7 +374,7 @@ GdiEngine::~GdiEngine()
         // Because the API is affected by the raster/TT status of the actively selected font, we can't have
         // GDI choosing a TT font for us when we ask for Raster. We have to settle for forcing the current system
         // Terminal font to load even if it doesn't have the glyphs necessary such that the APIs continue to work fine.
-        if (0 == wcscmp(FontDesired.GetFaceName(), L"Terminal"))
+        if (FontDesired.GetFaceName() == L"Terminal")

```

Comment:
I know it didn't have a whole lot to do with you here, but should the magic string `L"Terminal"` be a constexpr at the top instead of here?